### PR TITLE
Migrate terra standout

### PR DESCRIPTION
**Pull request by sid141288**
### Summary
Migration of terra-standout from old repo to the mono-repo.
screen shot 2017-03-07 at 4 46 16 pm

### CODE REVIEW

File: packages/terra-standout/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{TBD}
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-standout`
+  - `yarn add terra-standout`
+
+## Usage
+Having text standout ca be achieved by assigning following two modifier classes to a text container.
```
Comment (bjankord) [inline packages/terra-standout/docs/README.md]: typo `ca` => `can`

File: packages/terra-standout/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
```
Comment (bjankord) [inline packages/terra-standout/package.json]: Lets remove `react` and `react-dom` from the peerDependencies.
Also, we need to add `terra-mixins` to the peerDependencies.

e.g. 
```
"peerDependencies": {
  "terra-mixins": "1.0.0"
},
"dependencies": {
  "terra-mixins": "1.0.0"
},
```

File: packages/terra-standout/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rm -rf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "$(cd ..; npm bin)/stylelint src/**/*.scss",
+    "release:major": "npm test && node ../../scripts/release/release.js major",
+    "release:minor": "npm test && node ../../scripts/release/release.js minor",
+    "release:patch": "npm test && node ../../scripts/release/release.js patch",
+    "test": "npm run test:spec && npm run test:nightwatch-default",
```
Comment (bjankord) [inline packages/terra-standout/package.json]: Remove all the test scripts and replace them with the following:
```
"test": "echo \"Warning: no test written\" && exit 0"
```

We'll add jest and nightwatch tests as we iterate on this component.

File: packages/terra-standout/docs/README.md (hunk 1 above)
Comment (sid141288) [inline packages/terra-standout/docs/README.md]: Updated it. :)

File: packages/terra-standout/package.json (hunk 2 above)
Comment (sid141288) [inline packages/terra-standout/package.json]: Updated the dependencies. :)

File: packages/terra-standout/package.json (hunk 3 above)
Comment (sid141288) [inline packages/terra-standout/package.json]: Updated the scripts to display warning for the tests. :)

**Comment by Matt-Butler** [comment]
Nice job Sid

**Comment by sid141288** [comment]
@Matt-Butler Thank You :)

**Pull request merged**